ros_gz_sim_demos/worlds/ros_gz.sdf

```diff
@@ -70,7 +70,7 @@
 
     <plugin name="ros_gz_sim::ROSGzBridge"
             filename="ros-gz-bridge-system">
-      <config_file>package://ros_gz_sim_demos/config/full.yaml</config_file>
+      <config_file>package://ros_gz_bridge/config/full.yaml</config_file>
```

Comment:
I think you moved this file to `package://ros_gz_bridge/test/config/full.yaml`, isn't it ?

Reply:
Thanks, updated in [9576d86](https://github.com/gazebosim/ros_gz/pull/515/commits/9576d862e320d75253ec78457d1f4b59c00a83f4).